kiwi/boot/functions.sh

```diff
@@ -5525,18 +5540,25 @@ function waitForStorageDevice {
     # which could be a whole disk or a partition.
     # the function will wait until the size of the
     # storage device could be obtained or the check
-    # counter equals 4
+    # counter equals 30. For USB devices it waits
+    # only until check counter equals 2.
     # ----
     local IFS=$IFS_ORIG
     local device=$1
     local check=0
+    local transport="`getDeviceTransportType $device`"
+    if [[ $transport == *"usb"* ]];then
```

Comment:
ok so let's change this to a simple 

```bash
if [ "$transport" = "usb" ];then
```

in the worst case it will result in the default wait period which is ok for me. But if we wait too short for a device it would have a bad impact